Android/Wear/External-Dependency-Info.txt

```diff
@@ -0,0 +1,29 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for Google's Android Wear Libraries incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# Google Android Wear Libraries
+# https://developer.android.com/wear
+# https://dl.google.com/dl/android/maven2/com/google/android/support/wearable/2.2.0/wearable-2.2.0.pom
```

Comment:
I think it may be better to split this into two separate nodes, just in case.

Reply:
You mean list the license once for each library instead?  Sure, not a bad idea.